service/rds/rdsutils/example_test.go

```diff
@@ -0,0 +1,91 @@
+package rdsutils_test
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"os"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds/rdsutils"
+)
+
+// Example contains usage of assuming a role and using
+// that to build the auth token.
+// Usage:
+//	./main -user "iamuser" -dbname "foo" -region "us-west-2" -rolearn "arn" -endpoint "dbendpoint" -port 3306
+func ExampleRDSUtils_ConnectViaAssumeRole() {
```

Comment:
This example naming is causing the tests to fail.  To have the example paired with the godocs this should be named. `ExampleBuildAuthToken`